kernel/arch/dreamcast/include/arch/irq.h

```diff
@@ -401,7 +559,7 @@ typedef void (*irq_handler)(irq_t code, irq_context_t *context, void *data);
     \param  data            A pointer that will be passed along to the callback.
     
     \retval 0               On success.
-    \retval -1              If the code is invalid.
+    \retval -1              If the source is invalid.
```

Comment:
This should be consistent across the handler set/get/type. Here you changed the comment to note `source` but didn't change the function's param to be `source` and in set handler it's `code` in the doc but `source` in the param.

Reply:
Shit. Looks like it should've stayed `code`. Thanks.